content/200-orm/200-prisma-client/500-deployment/301-edge/550-deploy-to-deno-deploy.mdx

```diff
@@ -232,14 +232,6 @@ To set up a GitHub repository:
    git push -u origin main
    ```
 
-<Admonition>
```

Comment:
Should we instead update this Admonition to be for new PDP?

Reply:
Do you know if it's possible to connect a GitHub repo to the new PDP? I may be wrong but don't think this feature exists?

Reply:
AFAIK You can't connect a project to a new PDP project.

Reply:
Perfect, thanks for confirming Nurul!